src/components/config_profile/src/profile.cc

```diff
@@ -380,17 +384,19 @@ Profile::Profile()
     , remove_bundle_id_attempts_(kDefaultRemoveBundleIDattempts)
     , max_number_of_ios_device_(kDefaultMaxNumberOfiOSDevice)
     , wait_time_between_apps_(kDefaultWaitTimeBetweenApps)
-    , enable_app_launch_ios_(kDefaultEnableAppLaunchIOS) {
+    , enable_app_launch_ios_(kDefaultEnableAppLaunchIOS)
+    , error_occured_(false)
+    , error_description_() {
   // SDL version
   ReadStringValue(
       &sdl_version_, kDefaultSDLVersion, kMainSection, kSDLVersionKey);
 }
 
 Profile::~Profile() {}
 
-void Profile::config_file_name(const std::string& fileName) {
-  if (false == fileName.empty()) {
-    config_file_name_ = fileName;
+void Profile::set_config_file_name(const std::string& file_name) {
+  if (false == file_name.empty()) {
+    config_file_name_ = file_name;
```

Comment:
@LuxoftAKutsan Right. Nothing. As required. In this case SDL will take hardcoded default values.